ui-grid/ui-grid.d.ts

```diff
@@ -995,6 +999,12 @@ declare module uiGrid {
              */
             rowsVisibleChanged: (scope: ng.IScope, handler: rowsVisibleChangedHandler) => void;
             /**
+             * is raised after the sort criteria on one or more columns has changed
+             * @param {ng.IScope} scope Grid scope
+             * @param {rowsVisibleChangedHandler} handler callback
+             */
```

Comment:
Sure, it's here: http://ui-grid.info/docs/#/api/ui.grid.core.api:PublicApi - please look for sortChanged - I don't know why it is in Methods section rather than in Events, perhaps some problem in the documentation...